operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -94,13 +94,18 @@ spec:
 #  auth:
 #
 # Determines what authentication strategy to use when users log into Kiali.
-# Options are "login", "anonymous", "openshift".
+# Options are "login", "anonymous", "openshift", "token".
```

Comment:
ldap?

Reply:
yeah, while we are here, let's add the missing `ldap` , too. We must have forgot this spot when we added ldap support.

Reply:
Yes, I tried to add ldap in all places where it was omitted. I missed this one.
Thanks for pointing it.